### ISSUE TITLE
Are reference results available?

### ISSUE THREAD

**Issue by maltelenz** [christiankral/PhotoVoltaics#48]
As a tool developer, it would be of interest to have reference results/baselines for expected results available, for the simulateable models in the `PhotoVoltaics` library. This would allow for a tool vendor to verify that the library works correctly in the tool.

Are such results available in any form?

**Comment by christiankral** [comment]
I agree that it makes sense to provide reference signals. In the MSL we specify the reference signals in text files located in the `./Resources/Refrerence/...`  folder. I could specify reference signals in a similar way for the PhotoVoltaics library. 

As far as I remember @GallLeo was creating the actual results based on Dymola and a (Windows) script. The actual simuklation results are stored at [MAP-LIB_ReferenceResults](https://github.com/modelica/MAP-LIB_ReferenceResults) . But I do not know much about this procedure. Possibly @GallLeo can make a comment here.
